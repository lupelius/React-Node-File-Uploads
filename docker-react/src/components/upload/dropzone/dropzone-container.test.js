// search-container.test.js
import React from 'react';
import renderer from 'react-test-renderer';

test('Dropzone matches snapshot', () => {
  const openFileDialog = event => {};
  const onDragOver = event => {};
  const onDragLeave = event => {};
  const onDrop = event => {};
  const disable = event => {};
  const onFilesAdded = event => {};
  const component = renderer.create(
    <div 
      className="Dropzone"
      onClick={openFileDialog}
      onDragOver={onDragOver}
      onDragLeave={onDragLeave}
      onDrop={onDrop}
      style={{ cursor: disable ? "default" : "pointer" }}
    >
      <img
        alt="upload"
        className="Icon"
        src="upload.svg"
      />
      <input
        className="FileInput"
        type="file"
        multiple
        onChange={onFilesAdded}
      />
      <span>Drop</span>
    </div>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});