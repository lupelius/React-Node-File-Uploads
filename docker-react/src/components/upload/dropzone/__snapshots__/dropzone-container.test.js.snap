// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropzone matches snapshot 1`] = `
<div
  className="Dropzone"
  onClick={[Function]}
  onDragLeave={[Function]}
  onDragOver={[Function]}
  onDrop={[Function]}
  style={
    Object {
      "cursor": "default",
    }
  }
>
  <img
    alt="upload"
    className="Icon"
    src="upload.svg"
  />
  <input
    className="FileInput"
    multiple={true}
    onChange={[Function]}
    type="file"
  />
  <span>
    Drop
  </span>
</div>
`;
